packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -251,6 +251,34 @@ describe('build-schema', () => {
         expectValidJsonSchema(jsonSchema);
       });
 
+      it('properly converts nested array property when json schema provided', () => {
```

Comment:
Do we have any tests on querying/filtering the nested arrays?

Reply:
@agnes512 `@loopback/repository-json-schema` doesn't have it. If we add tests, need to do it in the acceptance tests for connectors.

Reply:
I mentioned it is because when I was triaging https://github.com/strongloop/loopback-connector-postgresql/issues/441, I realized that not all connectors support arrays/nested arrays. It'd be good if we can have them documented and tested. Just some thoughts 😄

Reply:
> I realized that not all connectors support arrays/nested arrays.

Yeah that's the problem, the original issue https://github.com/strongloop/loopback-next/issues/4754 is not trying to support the full feature, it's a fix for the misuse of decorator. 

> It'd be good if we can have them documented and tested.

It's not a trivial thing to test across connectors, let's do it in a separate story.